modules/internal/ChapelDomain.chpl

```diff
@@ -1377,9 +1367,9 @@ module ChapelDomain {
          addition should occur is unknown. We expect this to change in the
          future.
 
-       :arg inds: Indices to be added. ``inds`` can be an array of
-                  ``rank*idxType`` or an array of ``idxType`` for
-                  1-D domains.
+       :arg inds: Indices to be added. ``inds`` must be an array of
+                  ``rank*idxType``, except for 1-D domains it must be
```

Comment:
Wording idea: ", except for 1-D domains, where it must be an array of ``idxType``"